daemon/emer-daemon.c

```diff
@@ -910,94 +908,6 @@ handle_upload_finished (EmerDaemon *self)
 }
 
 static void
-update_timestamps (EmerDaemon *self)
-{
-  EmerDaemonPrivate *priv = emer_daemon_get_instance_private (self);
-
-  GError *error = NULL;
-  if (!emer_persistent_cache_get_boot_time_offset (priv->persistent_cache,
```

Comment:
Since this was the only call to `emer_persistent_cache_get_boot_time_offset` that passed `TRUE` for the `always_update_timestamps` parameter, we should also remove the `always_update_timestamps` parameter from `emer_persistent_cache_get_boot_time_offset`. This will discourage someone from reintroducing a call that passes `TRUE` to `always_update_timestamps`, which was something of a hack in the first place.